cmake/modules/hunter_config.cmake

```diff
@@ -44,9 +99,23 @@ macro(hunter_config)
 
   string(COMPARE NOTEQUAL "${_hunter_GIT_SUBMODULE}" "" _hunter_submodule_create)
   if(_hunter_submodule_create)
+    # get HUNTER_SUBMODULE_SOURCE_SUBDIR from CMAKE_ARGS
+    parse_cmake_args_for_keyword(
+      CMAKE_ARGS ${_hunter_CMAKE_ARGS}
+      KEYWORD HUNTER_SUBMODULE_SOURCE_SUBDIR
+      OUTPUT source_subdir
```

Comment:
We are in macro here so names should start with `_`, it should be `_source_subdir`.